content/en/blog/2023/egress-sni/index.md

```diff
@@ -0,0 +1,441 @@
+---
+title: "[WIP] Routing HTTPS/TLS traffic to arbitrary remote destinations"
+description: "A generic approach to set up egress gateways and use them to route traffic to a restricted set of target remote hosts dynamically, supporting wildcard domains."
```

Comment:
```suggestion
title: "Routing egress gateway traffic to wildcard destinations"
description: "A generic approach to set up egress gateways that can route traffic to a restricted set of target remote hosts dynamically, including wildcard domains."
```

Reply:
Shouldn't `Routing egress gateway traffic` be just `Routing egress traffic`?